kitsune/search/v2/signals/forums.py

```diff
@@ -0,0 +1,25 @@
+from django.db.models.signals import post_save, post_delete
+from django.dispatch import receiver
+from django.conf import settings
+from kitsune.search.v2.es7_utils import index_object, delete_object, index_objects_bulk
+from kitsune.forums.models import Thread, Post
+
+
+@receiver(post_save, sender=Thread)
+def handle_forum_thread_save(instance, **kwargs):
+    if settings.ES_LIVE_INDEXING:
```

Comment:
nit: shall we add a decorator in the sumo app to use instead of adding this `if` everywhere?

Reply:
Absolutely!